Add io.github.ychclone.blink

ychclone opened the pull request:
<!-- ⚠️  The submission PR must be against the `new-pr` branch ⚠️  -->

### Please confirm your submission meets all the criteria

<!-- Please replace each `[ ]` by `[X]` when the step is complete -->

- [x] Please describe your application briefly.
- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

hfiguiere commented on io.github.ychclone.blink.yml:
```diff
@@ -0,0 +1,30 @@
+id: io.github.ychclone.blink
+runtime: org.kde.Platform
+runtime-version: '6.6'
```
6.7 is out.

ychclone replied:
Ok, updated

hfiguiere commented:
This is missing

> Please describe your application briefly.

Also please answer the other questions. It's not a captcha.

hfiguiere commented on io.github.ychclone.blink.yml:
```diff
@@ -0,0 +1,30 @@
+    builddir: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DCMAKE_INSTALL_LIBDIR:PATH=/app/lib
```
```suggestion
```

You shouldn't ned this.

ychclone replied:
Ok, updated

hfiguiere commented on io.github.ychclone.blink.yml:
```diff
@@ -0,0 +1,30 @@
+      - type: archive
+        url: https://github.com/ychclone/blink/archive/refs/tags/1.9.1.tar.gz
+        sha256: 8e68a17582091d7d6b5199e6e0cea2c4dd6b1670144ab4ca0a3a7fec7352aa41
+command: blink
```
this line should be above modules

ychclone replied:
Ok, updated

ychclone commented:
> This is missing
> 
> > Please describe your application briefly.
> 
> Also please answer the other questions. It's not a captcha.

Blink code search is a GUI of live indexed grep for source code. Fuzzy suggestion in autocomplete. Files locator, search and replace. Switch different projects and start searching. Drag and drop of filenames to your favourite editor.

ychclone commented:
> This is missing
> 
> > Please describe your application briefly.
> 
> Also please answer the other questions. It's not a captcha.

I updated the checklist

bbhtt commented on qscintilla-lib-paths.patch:
```diff
@@ -0,0 +1,31 @@
+ INSTALLS += header
+ 
+-trans.path = $$[QT_INSTALL_TRANSLATIONS]
++trans.path = /app/share/qt5/translations
+ trans.files = qscintilla_*.qm
+ INSTALLS += trans
+ 
+-qsci.path = $$[QT_INSTALL_DATA]
++qsci.path = /app/share/qt5
+ qsci.files = ../qsci
+ INSTALLS += qsci
+ 
+-features.path = $$[QT_HOST_DATA]/mkspecs/features
++features.path = /app/lib/qt5/mkspecs/features
```
How are these Qt5 still? You're on a Qt6 runtime

ychclone replied:
Ok, updated

bbhtt commented:
bot, build io.github.ychclone.blink

bbhtt commented on io.github.ychclone.blink.yml:
```diff
@@ -0,0 +1,29 @@
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+    sources:
+      - type: archive
+        url: https://github.com/ychclone/blink/archive/refs/tags/1.9.1.tar.gz
```
I don't see a metainfo file in the archive https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#path-and-filename

The page above has everything you need to create one.

ychclone replied:
Hi, I've added the metainfo in https://github.com/ychclone/blink/archive/refs/tags/1.9.1.tar.gz,
folder /app/share/metainfo

ychclone replied:
Hi, I've updated the sha256

hfiguiere commented:
> I updated the checklist

But you never described the app as requested.

ychclone commented:
> > I updated the checklist
> 
> But you never described the app as requested.

Hi, I've added the metainfo in https://github.com/ychclone/blink/archive/refs/tags/1.9.1.tar.gz,
folder /app/share/metainfo

bbhtt commented:
bot, build io.github.ychclone.blink

hfiguiere commented:
> > > I updated the checklist
> > 
> > 
> > But you never described the app as requested.
> 
> Hi, I've added the metainfo in https://github.com/ychclone/blink/archive/refs/tags/1.9.1.tar.gz, folder /app/share/metainfo

Still not done

![image](https://github.com/flathub/flathub/assets/114441/588420d0-bbcb-4d80-aa2f-0cde8882ec35)
